units/systemd-machined.socket

```diff
@@ -0,0 +1,21 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Virtual Machine and Container Registration Service Socket
+Documentation=man:systemd-machined.service(8)
+DefaultDependencies=no
```

Comment:
this seems unnecessary. machined itself is a regular service, hence the socket for it should also be regular. i.e. please drop the DefaultDependencies=no here, and the manual deps below